tests/bundle/test_bundle_adjustment.py

```diff
@@ -23,18 +23,19 @@ class TestBundleAdjustmentOptimizer(unittest.TestCase):
     def setUp(self):
         super().setUp()
 
-        self.obj = BundleAdjustmentOptimizer()
+        output_reproj_error_thresh = 100
+        self.obj = BundleAdjustmentOptimizer(output_reproj_error_thresh)
 
         self.test_data = EXAMPLE_DATA
 
-    def test_simple_scene(self):
-        """Test the simple scene using the `run` API."""
+    # def test_simple_scene(self):
+    #     """Test the simple scene using the `run` API."""
```

Comment:
why are we commenting out this test?

Reply:
We are no longer packaging the factor graph error with the result. So we would need to recompute that error, or save it in the `GtsfmData` class somehow. Any thoughts about that?

Reply:
got it. Yeah for now we can skip the test.